dev-bin/release.sh

```diff
@@ -44,7 +44,7 @@ fi
 php composer.phar self-update
 php composer.phar update --no-dev
 
-perl -pi -e "s/(?<=const VERSION = ').+?(?=';)/$tag/g" src/MinFraud.php
+perl -pi -e "s/(?<=const VERSION = ').+?(?=';)/$tag/g" src/ServiceClient.php
```

Comment:
This needs to be updated now.

Reply:
Thanks -- great catch! I've rebased this in with the initial abstraction commit.